src/main/kotlin/com/kotlin/boardproject/global/handler/TokenAccessDeniedHandler.kt

```diff
@@ -1,12 +1,12 @@
-package com.kotlin.boardproject.common.handler
+package com.kotlin.boardproject.global.handler
 
-import com.kotlin.boardproject.common.enums.ErrorCode
+import com.kotlin.boardproject.global.enums.ErrorCode
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end